src/Maestro/maestro-angular/src/maestro-client/maestro.ts

```diff
@@ -80,8 +80,8 @@ export class MaestroService {
 })
 export class MaestroModule {
     public static defaultOptions: MaestroOptions = {
-        baseUrl: "https://maestro-prod.westus2.cloudapp.azure.com/",
```

Comment:
Why mess around with the whitespace here? The before seems better than the after...

Reply:
Not intentionally, it happened as part of a revert.